src/pages/Calc.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Calculator from '../components/Calculator';
+import './Calc.css';
+
+function Calc() {
+  return (
+    <div className="calc-holder">
+      <h2 className="sub-title">Let`s do some math!</h2>
+      <Calculator />
+    </div>
+  );
+}
```

Comment:
[OPTIONAL]
- I would recommend using `ES6` arrow function instead of the `function` keyword. It is a cleaner way to write a function and also has many benefits, you can check that in this [article](https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/).